src/vm/docs/mod.rs

```diff
@@ -205,6 +195,18 @@ fn make_for_simple_native(api: &SimpleFunctionAPI, function: &NativeFunctions) -
     }
 }
 
+const EQUALS_API: SpecialAPI = SpecialAPI {
+    name: "eq?",
+    input_type: "A, A, ...",
+    output_type: "bool",
+    signature: "(eq? v1 v2...)",
+    description: "Compares the inputted values, returning `true` if they are all equal. Note that _unlike_ the `(and ...)` function, `(eq? ...)` will _not_ short-circuit.",
```

Comment:
Ah, good point.  Do we want `eq?` to short-circuit?

Reply:
Hmm, it doesn't usually in Lisps, and I'm inclined to follow that convention.

Reply:
Okay, sounds good.  Let's just follow the convention (but thanks for mentioning it explicitly here).